region/brinstar/red/Red Tower.json

```diff
@@ -1435,52 +1432,112 @@
       "requires": [
         "SpaceJump",
         {"or": [
-          "canCarefulJump",
-          "HiJump"
+          "HiJump",
+          {"and": [
+            "canCarefulJump",
+            "canWalljump"
+          ]}
         ]},
-        {"or": [
-          {"enemyKill": {
-            "enemies": [["Ripper", "Ripper", "Ripper", "Ripper"]]
-          }},
-          {"obstaclesCleared": ["B"]}
-        ]}
+        {"obstaclesCleared": ["B"]}
       ],
-      "clearsObstacles": ["B"],
       "note": [
-        "This strat is for killing all of the Rippers and then using Space Jump to get up without wall jumps.",
+        "This strat is for killing all of the Rippers and then using Space Jump to get up without consecutive wall jumps.",
         "Space jump up near the shot block, then shoot it while falling right as it goes off screen, then space jump back up."
       ]
     },
     {
       "link": [9, 5],
-      "name": "Red Tower IBJ",
+      "name": "Tricky Space Jump",
       "requires": [
-        "h_canIBJ",
+        "SpaceJump",
+        "canTrickyJump",
+        {"obstaclesCleared": ["B"]}
```

Comment:
A tricky space jump could shoot the shot block out while jumping upwards, and doesn't need to clear the rippers.  Its a little hard for very hard, but they could do it.  And Hijump helps a bit.